.gitignore

```diff
@@ -1 +1,2 @@
 /target
+myfile.torrent
```

Comment:
This is the file used for writing my own torrent file based on reading and parsing the original file's contents. It's probably an integration test.